.template-lintrc.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+const pending = [
+];
+
 module.exports = {
-  extends: 'recommended'
+  extends: 'octane',
+  pending
```

Comment:
what's this pending thing?

Reply:
`pending` is a list of violations that haven't been resolved. I probably should have removed this given that there are none. Historically I've kept this in my apps/addons to make future diffs easier to read when new violations pop up.